src/Uno.UI/UI/Xaml/Controls/ContentPresenter/ContentPresenter.cs

```diff
@@ -724,6 +724,8 @@ protected override void OnLoaded()
 			SynchronizeContentTemplatedParent();
 
 			UpdateBorder();
+
+			SetImplicitContent();
```

Comment:
should this be called before `SetUpdateTemplate()`?

Reply:
Has been moved to OnTemplatedParentChanged.